test/addons-abi/test_instanceof/test_instanceof.cc

```diff
@@ -28,10 +28,9 @@ void Init(napi_env env, napi_value exports, napi_value module) {
     { "doInstanceOf", doInstanceOf },
   };
 
-  for (int i = 0; i < sizeof(descriptors) / sizeof(*descriptors); i++) {
-    status = napi_define_property(env, exports, &descriptors[i]);
-    if (status != napi_ok) return;
-  }
+  status = napi_define_properties(
+    env, exports, sizeof(descriptors) / sizeof(*descriptors), descriptors);
+  if (status != napi_ok) return;
```

Comment:
In all the test code I have kept the consistent pattern of checking the status after a NAPI call, even if there are no statements following. It makes it easier to copy-paste and refactor code without accidentally forgetting the error-checking.